bundles/core/src/main/java/com/adobe/cq/email/core/components/ajo/impl/EmailContentExporterImpl.java

```diff
@@ -0,0 +1,117 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2022 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+package com.adobe.cq.email.core.components.ajo.impl;
+
+import java.io.IOException;
+
+import org.apache.http.HttpHeaders;
+import org.apache.http.HttpStatus;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.metatype.annotations.AttributeDefinition;
+import org.osgi.service.metatype.annotations.Designate;
+import org.osgi.service.metatype.annotations.ObjectClassDefinition;
+
+import com.adobe.cq.email.core.components.ajo.AjoException;
+import com.adobe.cq.email.core.components.ajo.EmailContentExporter;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+@Component(service = EmailContentExporter.class)
+@Designate(ocd = EmailContentExporterImpl.Config.class)
+public class EmailContentExporterImpl implements EmailContentExporter {
+
+    public static final String TEMPLATES_ENDPOINT = "https://platform-stage.adobe.io/journey/authoring/message/templates";
+
+    @ObjectClassDefinition(
+        name = "Export To AJO",
+        description = "Export To AJO"
+    )
+    @interface Config {
+        @AttributeDefinition(
+            name = "API Key",
+            description = "API Key"
+        )
+        String apiKey() default "cjm-authoring-ui";
+
+        @AttributeDefinition(
+            name = "Organization ID",
+            description = "Organization ID"
+        )
+        String orgId() default "745F37C35E4B776E0A49421B@AdobeOrg";
```

Comment:
Is this just temporary right? not sure if everybody knows what org it represents